common/src/main/java/com/wynntils/features/chat/ChatTabsFeature.java

```diff
@@ -112,8 +113,8 @@ public void onChatScreenKeyTyped(ChatScreenKeyTypedEvent event) {
         Managers.ChatTab.setFocusedTab(Managers.ChatTab.getNextFocusedTab());
     }
 
-    @Override
-    protected void postEnable() {
+    @SubscribeEvent
+    public void onEnable(FeatureStateChangeEvent.Enabled event) {
         Managers.ChatTab.resetFocusedTab();
```

Comment:
This will reset the focused tab any time any feature whatsoever gets enabled...

Reply:
Fixed with the "revert"